src/readinglistbar.cpp

```diff
@@ -37,7 +40,6 @@ ReadingListBar::ReadingListBar(QWidget *parent) :
     auto importAction = app->getAction(KiwixApp::ImportReadingListAction);
     connect(exportAction, &QAction::triggered, this, &ReadingListBar::onExport);
     connect(importAction, &QAction::triggered, this, &ReadingListBar::onImport);
-    ui->label->setText(gt("reading-list-title"));
```

Comment:
So this was replaced with `ui->titleLabel->setText(gt("reading-list"));` (and moved to the beginning of the constructor body). The new name of the label data member makes more sense, however it's not clear why the i18n text string id was changed from `"reading-list-title"` to `"reading-list"`.

But checking `resources/i18n/qqq.json` and `resources/i18n/en.json` raises another question - do we really need both those strings? One of them is used as the tooltip for the reading list button and the other as the heading of the reading list bar. In English translation file the difference between them is only in capitalization, while their descriptions in `qqq.json` are identical. We should either drop one of them, or describe them differently in `qqq.json`.

@ShaopengLin @kelson42

Reply:
Hello! thank you for reviewing.. the reason i changed it from `"reading-list-title"` to `"reading-list"` is to maintain consistency with code present in tableofcontentbar.cpp😅 and since it worked i didn't got into checking the difference it might cause...but now that you have investigated about these strings, let me know how we should proceed further...drop one of them or change their description.

Reply:
@kelson42 Will you please provide your feedback to my comment above?

Reply:
@veloman-yunkan Nomstrong opinion, but if I has to choose I would say that having two strings seems wrong.

Reply:
@heropj Then let's get rid of the `"reading-list-title"` and use `"reading-list"` instead. But I think that its capitalization must be changed to title case.

Reply:
@veloman-yunkan Do i need to remove `"reading-list-title"` from all translation files?? or just from `qqq.json` and `en.json`??

Reply:
Only `qqq.json` and `en.json` must be updated.